### PR TITLE
  feat: include tx_hash on NameClaim activity

### PR DESCRIPTION
**Pull request by sborrazas**

### CODE REVIEW

**Review by thepiwo — APPROVED** [review]
can we discuss a way were these model changes are consistent across endpoints?

**Comment by sborrazas** [comment]
> can we discuss a way were these model changes are consistent across endpoints?

this is very much related to the `unify reference of transactions internally` feature. I think these two fields we're adding here (`tx` and `tx_hash`) are going to be present anyway. but when implementing that feature we should discuss ways in which we can expose whether the `tx_hash` points to a contract call tx or an actual name claim

File: test/ae_mdw_web/controllers/activities_controller_test.exs [hunk 1]
```diff
@@ -767,29 +767,33 @@ defmodule AeMdwWeb.ActivitiesControllerTest do
                  "height" => ^height2,
                  "type" => "NameClaimEvent",
                  "payload" => %{
```
Comment (jyeshe) [inline test/ae_mdw_web/controllers/activities_controller_test.exs:769]: Could we have also the other two added fields, specially the `tx_hash`?

File: test/ae_mdw_web/controllers/activities_controller_test.exs [hunk 2]
```diff
@@ -767,29 +767,35 @@ defmodule AeMdwWeb.ActivitiesControllerTest do
...
-                   "account_id" => ^account,
-                   "fee" => 333_333,
-                   "name" => ^plain_name,
-                   "name_fee" => 33333,
-                   "name_salt" => 3333,
-                   "nonce" => 333,
-                   "ttl" => 3_333_333,
-                   "type" => "NameClaimTx"
+                   "tx_hash" => "th_2CdVYuqtpcjoshDw2otjbLEyj8SpxjMP9MCgpn1oU9zGaqvUn4",
```
Comment (jyeshe) [inline test/ae_mdw_web/controllers/activities_controller_test.exs:770]: can we pattern match please to avoid looking up for the hash?

Comment (jyeshe) [reply test/ae_mdw_web/controllers/activities_controller_test.exs:770]: The `micro_time` could be easily tested the same way please.

Comment (sborrazas) [reply test/ae_mdw_web/controllers/activities_controller_test.exs]: Done

**Review by jyeshe — APPROVED** [review]

**Pull request merged**